db/people.yml

```diff
@@ -552,7 +552,7 @@ stao:
     - Tao, Songsheng
     - S. Tao
     - Tao, S.
-  avatar: ''
+  avatar: https://avatars0.githubusercontent.com/u/37090959?s=400&u=a94748131d06d93dfcaa6144e497870755cd2fea&v=4
   bio: ''
```

Comment:
Although you can put an empty string in, you might consider writing a small blurb about yourself (it helps when people search for you and such).

Reply:
Songsheng, this will appear in your web-profile when we move our website to this platform.